package.json

```diff
@@ -2,13 +2,23 @@
   "name": "taiko-diagnostics",
   "version": "0.0.1",
   "description": "A plugin for taiko which provides some diagnostics features",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "build": "babel src -d lib"
+  },
+  "jest": {
+    "verbose": true
   },
   "author": "BugDiver",
   "license": "MIT",
   "dependencies": {
-    "speedline": "^1.4.2"
+    "@babel/core": "^7.3.4",
+    "babel-jest": "^24.3.1",
+    "jest": "^24.3.1",
+    "speedline": "^1.4.2",
+    "@babel/cli": "^7.2.3",
+    "@babel/preset-env": "^7.3.4",
+    "taiko": "git://github.com/getgauge/taiko.git"
```

Comment:
@saikrishna321 Shouldn't all these dependencies are required only for development?
We only need `speedline` for as production dependency.
Could you please move them to `dev-dependencies`?